web/src/hotkey.js

```diff
@@ -320,17 +319,6 @@ export function process_escape_key(e) {
             return true;
         }
 
-        if ($("#searchbox").has(":focus")) {
-            $("input:focus,textarea:focus").trigger("blur");
-            if (page_params.search_pills_enabled) {
-                $("#searchbox .pill").trigger("blur");
-                $("#searchbox #search_query").trigger("blur");
-            } else {
-                message_view_header.exit_search();
```

Comment:
This looks like it removes something that does stuff, but I think we're using this hook in `search.js` instead:
```
        // Use our custom typeahead `on_escape` hook to exit                                            
        // the search bar as soon as the user hits Esc.                                                 
        on_escape: message_view_header.exit_search,                                                     

```

Reply:
There was no `$("#searchbox")`, so this could not have done stuff.